presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -163,7 +163,7 @@ private static AWSGlueAsync createAsyncGlueClient(GlueHiveMetastoreConfig config
             asyncGlueClientBuilder.setRegion(config.getGlueRegion().get());
         }
         else if (config.getPinGlueClientToCurrentRegion()) {
-            Region currentRegion = Regions.getCurrentRegion();
+            Region currentRegion = AwsCurrentRegionHolder.getCurrentRegionFromEC2Metadata();
```

Comment:
Nit: we can static import these since the method name is self-descriptive.

Reply:
Will update.